index.html

```diff
@@ -1458,11 +1462,26 @@ <h2 id="OfflineAudioContext">
                   start rendering <code>length</code> sample-frames of audio
                   into <var>buffer</var>.
                   </li>
-                  <li>For every render quantum, check and suspend the rendering
-                  if necessary.
+                  <li>At the beginning of every render quantum, check scheduled
```

Comment:
This would be better put inside the section "processing model" when my branch merges.